tests/openthread/README.md

```diff
@@ -1,36 +1,45 @@
 ## OpenThread on RIOT
 
-This test demonstrates the [OpenThread](https://github.com/openthread/openthread) stack running on RIOT. When flashed,
-it will initialize the OpenThread Command Line Interface for interacting with the stack.
+This test demonstrates how to use OpenThread stack from [OpenThread](https://github.com/openthread/openthread) on
```

Comment:
I would rewrite the first phrase like this:
This test demonstrates how to use the [OpenThread](https://github.com/openthread/openthread) open source implementation of [Thread](https://threadgroup.org/).